modules/node/src/appInstance/appInstance.entity.ts

```diff
@@ -124,3 +118,99 @@ export class AppInstance<T extends AppName = any> {
   @UpdateDateColumn()
   updatedAt: Date;
 }
+
+export interface AppInstanceJSON<T extends AppName = any> {
```

Comment:
I learned that it is possible to instantiate a TypeORM entity without connecting to the database, you just have to manually assign all of the entity's properties. The `*JSON` interfaces defined here and elsewhere represent a JSON-compatible version of the database entity, so that it can be put in Redis properly.

Reply:
We have a standard type exported from the `types` module for `AppInstanceJSON` and `StateChannelJSON`. Can we try to reuse those types? They are standard across node and client.